pkg/setting/setting.go

```diff
@@ -819,6 +820,9 @@ func (cfg *Cfg) loadSpecifiedConfigFile(configFile string, masterFile *ini.File)
 		return fmt.Errorf("failed to parse %q: %w", configFile, err)
 	}
 
+	// micro-optimization since we don't need to share this ini file. In
+	// general, prefer to leave this flag as true as it is by default to prevent
+	// data races
```

Comment:
what preventing us from setting it `true`?

Reply:
Nothing at all, I can remove it and move the comment to another place if you think that would be better or more consistent. It's just that I noticed with git blame that it was several folks who set the flag to false, so I thought it would be good to leave this one and explain why it's ok here and not in general. That with the hope that if some of them come back and want to set another instance to false then they would probably look up what was the name of the frobb name to be changed in the code, see the comment and reconsider.

Reply:
Merging the PR. If you think it would be better to remove this line as well, let me know and I can open a followup PR.